flink-table/flink-table-common/src/main/java/org/apache/flink/table/functions/BuiltInFunctionDefinitions.java

```diff
@@ -759,6 +760,24 @@
                                             explicit(DataTypes.STRING()))))
                     .build();
 
+    /**
+     * Special "+" operator used internally by {@code SumAggFunction} to implement SUM aggregation
+     * on a Decimal type. Uses the {@link LogicalTypeMerging#findSumAggType(LogicalType)} to avoid
+     * the normal {@link #PLUS} override the special calculation for precision and scale needed by
+     * SUM.
+     */
+    public static final BuiltInFunctionDefinition PLUS_AGG =
+            BuiltInFunctionDefinition.newBuilder()
+                    .name("PLUS_AGG")
```

Comment:
`PLUS_AGG` looks like is an aggregate function..Maybe `AGG_PLUS`、`PLUS_FOR_AGG`..

Reply:
I also think it looks like an aggregation function. `AGG_DECIMAL_PLUS`?